accounts/abi/abi_test.go

```diff
@@ -689,7 +689,11 @@ func TestABI_MethodById(t *testing.T) {
 	}
 	for name, m := range abi.Methods {
 		a := fmt.Sprintf("%v", m)
-		b := fmt.Sprintf("%v", abi.MethodById(m.Id()))
+		m2, err := abi.MethodById(m.Id())
+		if err != nil {
+			t.Fatal(err)
```

Comment:
And this is where you append errors :)

`t.Fatalf("Failed to look up ABI method: %v", err)`